src/fpti.js

```diff
@@ -57,7 +57,8 @@ export const FPTI_KEY = {
     PAY_NOW:                   ('pay_now' : 'pay_now'),
     STICKINESS_ID:             ('stickiness_id' : 'stickiness_id'),
     TIMESTAMP:                 ('t' : 't'),
-    OPTION_SELECTED:           ('optsel' : 'optsel')
+    OPTION_SELECTED:           ('optsel' : 'optsel'),
+    USER_IDENTIFIER_MECHANISM: ('user_identifier_mechanism' : 'user_identifier_mechanism')
```

Comment:
I'm cool with this name but wanted put another option out there that's a bit easier to spell: `USER_IDENTITY_METHOD`

Reply:
Do we want to include the possible values in here as well? ex:
```js
export const USER_IDENTIFIER_MECHANISM_VALUE = {
    LOCAL_STORAGE: ('local_storage' : 'local_storage'),
    COOKIE: ('cookie' : 'cookie'),
    CLIENT_TOKEN: ('client_token' : 'client_token')
}
```

Reply:
1. I would prefer to leave the key matching the actual value that we use in FPTI so that we don't lose out on the traceability of the code (it's confusing when fields are renamed). Do you happen to know if there's flexibility in that? otherwise I'll reach out to that team

Reply:
(sorry accidentally hit ctrl-shift and created a new comment)
2. Right now, we're only intending to make this request inside the smart-payment-buttons repo and the intention is that we may add additional methods going forward, so we want it to be pretty flexible. Since we're only using them in a single repo, I'm thinking I'll define the constants there and if we ever move it out of that repo we can add it to the shared constants repo?

Does that seem reasonable to you?

Reply:
> I would prefer to leave the key matching the actual value that we use in FPTI so that we don't lose out on the traceability of the code (it's confusing when fields are renamed). Do you happen to know if there's flexibility in that? otherwise I'll reach out to that team

If this is already an established field in FPTI then let's leave it as is (`USER_IDENTIFIER_MECHANISM`). I thought this was something brand new we were adding.

> 2. Right now, we're only intending to make this request inside the smart-payment-buttons repo and the intention is that we may add additional methods going forward, so we want it to be pretty flexible. Since we're only using them in a single repo, I'm thinking I'll define the constants there and if we ever move it out of that repo we can add it to the shared constants repo?

If it's only planned for SPB then I'm totally fine with the values living there. I originally thought we were going to test these from both paypal-checkout-components and SPB to validate 1st party cookies/local storage vs 3rd party.

Reply:
I think nothing is set in stone in FPTI yet; if you want to use a different key name just give Partheeban a heads up.

Reply:
Excellent, in that situation I definitely prefer @gregjopa's recommendation! Thanks y'all!